src/get-started/java-binding.md

```diff
@@ -781,6 +798,10 @@ To test API implemented with Vertx tools, use the tools described in the
 [project documentation](https://vertx.io/docs/vertx-junit5/java).
 You can use [Vertx Web Client][vertx-web-client] as a client or a different
 HTTP client.
+API can be tested with either unit tests with mocks of services (no TestKit
```

Comment:
I think the section is not coherent, it lacks structure: it starts with a suggestion to use a particular tool to test HTTP APIs, and then has an overview of various approaches to testing the API. 

Shan't it be the other way around:
1. The two approaches.
2. Unit tests with mocks.
3. Integration tests with Testkit.
4. Possible clients/extensions to use (or shall we skip it as the choice is rather wide — from numerous HTTP clients to specialized testing tools like REST-assured?)

Reply:
There are two ways to test API. First, API can be tested with unit tests by means of service mocks. TestKit is not required for these tests. Second, API can be tested with integration tests that use TestKit. For these tests TestKit provides the [`getPort()`][testkit-get-port] method that retrieves a TCP port to interact with the service REST API.

To test API implemented with Vertx tools, use the tools described in the 
[project documentation](https://vertx.io/docs/vertx-junit5/java). You can use any HTTP client at your discretion.